playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -199,6 +199,10 @@ EDXAPP_ENABLE_CREDIT_API: false
 EDXAPP_CUSTOM_COURSES_EDX: false
 
 EDXAPP_ENABLE_SYSADMIN_DASHBOARD: false
+
+# Enable unrestricted access course staff to grade downloads
+EDXAPP_ALLOW_COURSE_STAFF_GRADE_DOWNLOADS: true
```

Comment:
Set to a default value of false to be consistent with feature flag documentation: http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/feature_flags/feature_flag_index.html

Reply:
In that case we don't need this change as by default it is already set to false in edx-platform
https://github.com/Microsoft/edx-platform/blob/oxa/dev.fic/lms/envs/common.py#L208

In order to make it true, oxa-config will override the value using server-vars.yml
https://github.com/Microsoft/oxa-tools/blob/generate-grades/config/server-vars.yml#L104